uaclient/entitlements/repo.py

```diff
@@ -259,15 +259,37 @@ def setup_apt_config(self, silent: bool = False) -> None:
         :raise UserFacingError: on failure to setup any aspect of this apt
            configuration
         """
-        http_proxy = util.validate_proxy(
-            "http", self.cfg.apt_http_proxy, util.PROXY_VALIDATION_APT_HTTP_URL
-        )
-        https_proxy = util.validate_proxy(
-            "https",
-            self.cfg.apt_https_proxy,
-            util.PROXY_VALIDATION_APT_HTTPS_URL,
+        http_proxy: Optional[str] = None
```

Comment:
We can't type variables because we need to support Python 3.5. We can remove the typing, or if you want to make it explicit to mypy:
```suggestion
        http_proxy = None  # type: Optional[str]
```

Reply:
Oh yeah